composer.json

```diff
@@ -36,14 +36,16 @@
         "doctrine/orm": "~2.3",
         "symfony/yaml": "~2.2|~3.0",
         "symfony/validator": "~2.2|~3.0",
+        "symfony/property-info": "~2.8|~3.0",
         "symfony/phpunit-bridge": "~2.7|~3.0",
         "twig/twig": "~1.10",
         "satooshi/php-coveralls": "~0.6.1",
         "phpunit/phpunit": "~4"
     },
     "suggest": {
-        "symfony/web-profiler-bundle": "to use the data collector",
-        "doctrine/orm": "The Doctrine ORM integration is optional in the bundle."
+        "symfony/web-profiler-bundle": "To use the data collector.",
+        "doctrine/orm": "The Doctrine ORM integration is optional in the bundle.",
+        "symfony/property-info": "To use the PropertyInfo integration."
```

Comment:
I'm not sure this suggestion makes sense. Saying "install this component to use this component" looks weird. People wanting to use PropertyInfo will install it. And DoctrineBundle does not provide any feature based on the PropertyInfo which would let people wonder whether they want to use PropertyInfo. It only provides the inner parts for PropertyInfo.